test/Microsoft.Extensions.Logging.Analyzer.Test/Microsoft.Extensions.Logging.Analyzer.Test.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Add .NET Framework 4.6 to match the runtime used by VS when executing your analyzer.